dnf-docker-test/features/steps/repo_steps.py

```diff
@@ -224,12 +224,13 @@ def given_repository_with_packages(ctx, rtype, repository):
         ctx.text = template.render(name=name, **settings)
         fname = "{!s}/{!s}.spec".format(tmpdir, name)
         step_a_file_filepath_with(ctx, fname)
+        buildname = '%{NAME}-%{VERSION}-%{RELEASE}.%{ARCH}.rpm'
```

Comment:
Why should be epoch part of the rpm filename? At least Fedora and RHEL do not have epoch in the rpm filename.